test/specs/modules/Dropdown/Dropdown-test.js

```diff
@@ -994,6 +1011,7 @@ describe('Dropdown Component', () => {
         .simulate('focus')
 
       domEvent.keyDown(document, { key: 'ArrowDown' })
+      domEvent.keyDown(document, { key: 'ArrowDown' })
```

Comment:
There is a comment on 1008 that we should update or remove:

```js
// blur, focus, move item selection down
```

Since we are now opening after the focus.